strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/DefaultExceptionHandler.java

```diff
@@ -160,7 +160,7 @@ private MediaType requestedContent(WebRequest request)
         }
         catch (HttpMediaTypeNotAcceptableException e1)
         {
-            logger.error(String.format("Reuqested invalid content-type [%s]", request.getHeader(HttpHeaders.ACCEPT)));
+            logger.error(String.format("Requested invalid content-type [%s]", request.getHeader(HttpHeaders.ACCEPT)));
```

Comment:
Please, declare first a new `private final Logger logger = LoggerFactory.getLogger(DefaultExceptionHandler.class);` instead of using the parent one, which is not a logger from `org.slf4j` package.
And then change this snippet accordingly.